cmd/support-profile.go

```diff
@@ -114,22 +114,20 @@ func checkAdminProfileSyntax(ctx *cli.Context) {
 // working directory is a different partition. To allow all situations to
 // be handled appropriately use this function instead of os.Rename()
 func moveFile(sourcePath, destPath string) error {
-	inputFile, err := os.Open(sourcePath)
-	if err != nil {
-		return err
+	inputFile, e := os.Open(sourcePath)
+	if e != nil {
+		return e
 	}
+	defer inputFile.Close()
```

Comment:
I am not sure in Windows if we can move a file which is open, /cc @klauspost

Reply:
Good spot. It must be closed before delete can happen.